lib/northbound.c

```diff
@@ -1915,18 +2123,21 @@ const char *nb_err_name(enum nb_error error)
 	case NB_ERR_LOCKED:
 		return "resource is locked";
 	case NB_ERR_VALIDATION:
-		return "validation error";
+		return "validation";
 	case NB_ERR_RESOURCE:
 		return "failed to allocate resource";
 	case NB_ERR_INCONSISTENCY:
 		return "internal inconsistency";
 	default:
-		return "unknown";
+		snprintf(buf, sizeof(buf), "unknown(%u)", error);
+		return buf;
 	}
 }
 
 const char *nb_client_name(enum nb_client client)
 {
+	static char buf[32];
```

Comment:
nit: better to use macro than 32 as it is being used at multiple places.

Reply:
I removed these static buffers since they shouldn't be necessary in practice.